flaskbb/utils/helpers.py

```diff
@@ -222,7 +241,7 @@ def forum_is_unread(forum, forumsread, user):
         return False
 
     # check if the last post is newer than the tracker length
-    if forum.last_post_created < read_cutoff:
+    if not forum.last_post or forum.last_post_created < read_cutoff:
```

Comment:
This will introduce a new query for every forum that is displayed in a category or on the index page. Changing it to ``forum.last_post_id`` is not as nice but wont introduce any new queries (otherwise we could use ``forum.last_post.created``, etc.).